src/components/VioletButton/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './index.module.scss';
+
+function VioletButton({ text, typeButton, disabled }) {
+  return (
+    <div className={disabled ? styles.containerBtnDisabled : styles.containerBtnActive}>
+      {typeButton === 'button' ? (
+        <button
+          type="button"
+          className={disabled ? styles.btnDisabled : styles.btnActive}
+          disable={disabled.toString()}
```

Comment:
Cuidado con el typo acá:
```suggestion
          disabled={disabled.toString()}
```
Además de eso, es necesario pasar disabled a string?

Reply:
si no lo pasaba a string tiraba este error 
![image](https://user-images.githubusercontent.com/42507863/78932463-e635af00-7a7d-11ea-8b9b-2cc4915d0d6c.png)

Pero usando disabled y sin pasarlo a string  ya no da ese error